engine/runtime/src/test/java/org/enso/compiler/ExecCompilerTest.java

```diff
@@ -276,11 +276,12 @@ public void inlineReturnSignatureOnMemberMethod() throws Exception {
         ctx.eval(
             "enso",
             """
-    type My_Type
-        Value x
+            import Standard.Base.Data.Numbers
```

Comment:
this makes me sad a bit

Reply:
Yes, such a limitation in unbearable. We have to fix it in subsequent PRs. Reported as #8852